app/Services/AssignUserToOrganizationsService.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use App\Models\Organization;
+use App\Models\User;
+use Illuminate\Support\Facades\Http;
+
+class AssignUserToOrganizationsService
+{
+    public function assign(): void
```

Comment:
```suggestion
    public function assign(User $user): void
```

Reply:
Lets use `user` from `callback` method from `GithubController`

Reply:
changed assign to use user from callback method